<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget_Mycontent</class>
 <widget class="QWidget" name="Widget_Mycontent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>64</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_My" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>332</width>
     <height>64</height>
    </rect>
   </property>
   <widget class="PictureBox" name="myPic" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_My">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>96</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgba(102,102,102,1)</string>
    </property>
    <property name="text">
     <string>我的内容</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PictureBox</class>
   <extends>QWidget</extends>
   <header>picturebox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
